docs/pages/careers.tsx

```diff
@@ -183,6 +177,12 @@ const openRolesData = [
           'You will strengthen the advanced components team, build new ambitious complex features, work on strategic problems, and help grow the adoption.',
         url: '/company/react-engineer/',
       },
+      {
+        title: 'React Engineer - X - Community support',
+        description:
+          "You will provide support, remove blockers and unwrap potential features from reported issues for the advanced components team. You will directly impact developers' satisfaction and success.",
+        url: '/company/support-engineer/',
+      },
       {
         title: 'Hustler Engineer - Store',
```

Comment:
```suggestion
        title: 'Product Engineer - E-commerce',
```
I don't think "Hustler" is paints the role in a positive light.

Reply:
e-commerce, I hear Shopify for online good. Maybe?

```suggestion
        title: 'Product Engineer - Store',
```